src/Resources/config/api_resources/resources/Wishlist.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="%bitbag_sylius_wishlist_plugin.model.wishlist.class%" shortName="Wishlist">
+        <operations>
+            <operation
+                name="admin_get_wishlists"
+                class="ApiPlatform\Metadata\GetCollection"
+                uriTemplate="/admin/wishlists"
+            >
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>shop:wishlist:read</value>
```

Comment:
To be consistent with Sylius, it would be good to add a plugin specific prefix here